<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionAreaEditDialog</class>
 <widget class="QDialog" name="DetectionAreaEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select area of detection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Only select the sky in your area of detection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>There should be no trees, telephone lines, etc. in your selection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelInfo">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1. Take a picture with the webcam</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="image"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonTakePicture">
       <property name="text">
        <string>Take picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConnect">
       <property name="text">
        <string>Connect points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save to file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
